src/compas/robots/resources/basic.py

```diff
@@ -68,8 +69,9 @@ def can_load_mesh(self, url):
         #  - a one-letter scheme in Windows
         #  - file scheme
         is_local_file = len(scheme) in (0, 1) or scheme == 'file'
+        is_existing_file = os.path.isfile(url)
```

Comment:
I think this is the wrong place to check this. If `is_local_file` is False, we should not even try to check for file existence so I would suggest moving the check inside the `if` block (I know that now it looks pretty because all vars are assigned in one block but still).

Reply:
Something more like this?

```suggestion
is_local_file = len(scheme) in (0, 1) or scheme == 'file'

if is_local_file: 
    if os.path.isfile(url):
        return True
```